### ISSUE TITLE
Add URL scheme support

### ISSUE THREAD

**Issue by timonus** [gitpoint/git-point#733]

**Comment by SammyIsra** [comment]
Would [Deep Linking](https://medium.com/react-native-training/deep-linking-your-react-native-app-d87c39a1ad5e) be what we are looking for?

**Comment by timonus** [comment]
Yeah, that would probably be sufficient. I need to be able to fire a URL with an app-specific scheme like `gitpoint://` that can open particular GitHub entities.

**Comment by timonus** [comment]
Excellent, thanks!
